### PR TITLE
Estonia blog post and image

### PR DESCRIPTION
**Pull request by emileighoutlaw**
This blog post is prepped and ready for review.

### CODE REVIEW

**Comment by gboone** [comment]
You have two photos in here, did you mean to include both?

**Comment by emileighoutlaw** [comment]
Ah, no. One of them is the pre-ImageOptim image.

File: _posts/2015-06-17-Estonia.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+The European country has — since regaining its independence in 1991 —
+established itself as a leader in digital services. [The
+Atlantic](http://www.theatlantic.com/international/archive/2014/01/lessons-from-the-worlds-most-tech-savvy-government/283341/)
+called it, “the world’s most tech-savvy government.” Last week, a group
+of Estonia’s most prominent technologists visited D.C. for a week-long
+tour, which included meetings with the State Department, the World Bank,
+and U.S. digital services teams.
+
+<img src="/assets/blog/estonia/estonia2.jpg" width="500">
```
Comment (gboone) [inline _posts/2015-06-17-Estonia.md]: this should be ``

File: _posts/2015-06-17-Estonia.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+author:
+- emileigh
+tags:
+- usds
+- event
+excerpt: "At 18F, we were thrilled to meet for a discussion on digital services,
+cyber security, and IT funding with Estonia's most prominent technologists."
+description: "At 18F, we were thrilled to meet for a discussion on digital services,
+cyber security, and IT funding with Estonia's most prominent technologists."
+image: /assets/blog/Estonia/Estonia.jpg
```
Comment (gboone) [inline _posts/2015-06-17-Estonia.md]: This should be either `Estonia.jpg` or `Estonia2.jpg` depending on which we keep.

File: _posts/2015-06-17-Estonia.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+title: "What Estonia can teach us about the future of government records"
+date: 2015-06-17
+layout: post
+author:
+- emileigh
+tags:
+- usds
+- event
```
Comment (gboone) [inline _posts/2015-06-17-Estonia.md]: sorry, one more: this should be `events` maybe also add a tag for "estonia" or some way of capturing international collaboration?

**Comment by gboone** [comment]
This is looking good to me, let's wait for Travis and proof on staging.

**Pull request merged**